src/Sessions.cc

```diff
@@ -681,19 +681,23 @@ bool NetSessions::WantConnection(uint16_t src_port, uint16_t dst_port,
 
 void NetSessions::Weird(const char* name, const Packet* pkt, const char* addl)
 	{
+	std::string weird_name = name;
```

Comment:
Is this intermediate `std::string` generally needed?  Seems preferable to avoid allocation/copying if not.